File: platforms/WindowsPhone7/Accounter/Core/Record.cs
using System;
using System.Collections.Generic;

namespace Accounter.Core
{
	public class Record
	{
		private List<Cell> _cells = new List<Cell>();

		public String Code { get; set; }

		public List<Cell> Cells
		{
			get { return _cells; }
			set { _cells = value; }
		}

	}
}

File: platforms/Android/Accounter/src/com/vimukti/accounter/result/Record.java
package com.vimukti.accounter.result;

import java.util.ArrayList;
import java.util.List;

public class Record {
	private String code;
	private List<Cell> cells = new ArrayList<Cell>();

	public void setCells(List<Cell> cells) {
		this.cells = cells;
	}

	public List<Cell> getCells() {
		return cells;
	}

	public void setCode(String code) {
		this.code = code;
	}

	public String getCode() {
		return code;
	}
}
